packages/esm-patient-test-results-app/src/grouped-timeline/grouped-timeline.tsx

```diff
@@ -28,9 +28,23 @@ const PanelNameCorner: React.FC<PanelNameCornerProps> = ({ showShadow, panelName
   <TimeSlots className={`${styles.cornerGridElement} ${showShadow ? styles.shadow : ''}`}>{panelName}</TimeSlots>
 );
 
-const NewRowStartCell = ({ title, range, units, conceptUuid, shadow = false }) => {
+const NewRowStartCell = ({ title, range, units, conceptUuid, shadow = false, isString = false }) => {
   const { patientUuid } = usePatient();
 
+  if (isString) {
+    <div
+      className={styles.rowStartCell}
+      style={{
+        boxShadow: shadow ? '8px 0 20px 0 rgba(0,0,0,0.15)' : undefined,
+      }}
+    >
+      <span className={styles.trendlineLink}>{title}</span>
```

Comment:
It looks to me like the only difference is the content of the `div`. It seems to be copied otherwise - maybe we can actually change it to be either the `ConfigurableLink` with content or this `span` with content instead?